src/moditect/module-info.java

```diff
@@ -1,14 +1,13 @@
 // Generated 08-Mar-2019 using Moditect maven plugin
 module com.fasterxml.jackson.databind {
-    requires java.desktop;
-    requires java.logging;
+    requires static java.desktop;
```

Comment:
Maybe add a comment about reason for inclusion (java.beans.ConstructorProperties)?